acbuild/end.go

```diff
@@ -44,16 +46,13 @@ func runEnd(cmd *cobra.Command, args []string) (exit int) {
 		cmd.Usage()
 		return 1
 	}
-	if len(args) > 1 {
-		stderr("end: incorrect number of arguments")
-		return 1
-	}
 
 	if debug {
 		stderr("Ending build. Writing completed ACI to %s", args[0])
 	}
 
-	err := lib.End(tmpacipath(), args[0], path.Join(contextpath, workprefix), overwrite)
+	err := lib.End(tmpacipath(), args[0], path.Join(contextpath, workprefix),
+		overwrite, sign, args[1:])
```

Comment:
Yes it's intentional, and I want to expose it to users. I just need to document it better.

All arguments given to the end command after the path to the ACI to write are passed to gpg. This allows for the following:

```
acbuild end --sign thing3.aci -- --no-default-keyring --keyring test1.gpg
```